iib/workers/tasks/build.py

```diff
@@ -158,34 +158,41 @@ def _finish_request_post_build(
         index image.
     :param str overwrite_from_index_token: the token used for overwriting the input
         ``from_index`` image.
+    :return: pull spec of the index_image
+    :rtype: str
     :raises IIBError: if the manifest list couldn't be created and pushed
     """
     conf = get_worker_config()
+    state_reason = ''
```

Comment:
I'd prefer if this was removed since I'd rather get an exception in the tests if a code path does not define `state_reason`.

Reply:
yup, makes sense